requirements.txt

```diff
@@ -4,5 +4,6 @@ flask-oauthlib==0.9.3
 flask-restful==0.3.5
 flask-wtf==0.12
 flask==0.11.1
+httplib2==0.9.2
```

Comment:
Why not  ```httplib```/```urllib``` from [standard library](https://docs.python.org/release/2.7/library/httplib.html#examples)?